config/config_test.go

```diff
@@ -7,7 +7,7 @@ import (
 )
 
 func TestConfig(t *testing.T) {
-	Init("../meta-proxy.yml")
+	Init("yaml/local-meta-proxy.yml")
```

Comment:
```suggestion
	Init("meta-proxy-example.yml")
```

If I was a newbie of meta-proxy, I'm more willing to see the example config file in the project's root path, like `meta-proxy/meta-proxy-example.yml`,

or in `meta-proxy/config/meta-proxy-example.yml`.

Reply:
Ok, add `yaml` path is because we may have multi config and put all into `config` path is messy